File: Python/cars/UberX.py
from cars.Car import Car
class UberX(Car):
    __brand = str
    __model = str
    def __init__(self,license,driver,brand,model):
        super().__init__(license,driver)
        self.__brand = brand
        self.__model = model

    def setBrand(self,brand):
        self.__brand=brand
    
    def getBrand(self):
        return self.__brand

    def setModel(self,model):
        self.__model=model
    
    def getModel(self):
        return self.__model

File: PHP/cars/UberX.php
<?php
    require_once('Car.php');
    class UberX extends Car{
        private $brand;
        private $model;
    
        public function __construct($license,$driver,$brand,$model)
        {
            parent::__construct($license, $driver);
            $this->brand=$brand;
            $this->model = $model;
        }
        public function setModel($model){
            $this->model = $model;
        }
        public function setBrand($brand){
            $this->brand = $brand;
        }
    
        public function getModel(){
            return $this->model;
        }
        public function getBrand(){
            return $this->brand;
        }
        public function printDataCar(){
            parent::printDataCar();
        echo "brand: $this->brand \n
        model: $this->model \n
        
       
           ";
        }
            
        }



?>